src/gui/tray/UserModel.cpp

```diff
@@ -413,15 +413,30 @@ void User::slotAddError(const QString &folderAlias, const QString &message, Erro
     }
 }
 
+bool User::isValueableActivity(const Folder *folder, const SyncFileItemPtr &item) const
+{
+    // Check if we are adding it to the right account and if it is useful information (protocol errors)
+    const auto isDifferentAccount = folder->accountState() != _account.data();
+    const auto isConflictFromOriginalFile = item->_status == SyncFileItem::Conflict && !Utility::isConflictFile(item->_file);
+
+    if (isDifferentAccount) {
```

Comment:
@FlexW 

We still have too many lines of code IMHO.

`
bool User::isValueableActivity(const Folder *folder, const SyncFileItemPtr &item) const
{
    // activity for a different account - ignore it
    if (!folder->accountState() != _account.data()) {
        return false;
    }

    // first time conflict activity should be ignored until the next sync (fix conflict dialog crash issue)
    if (item->_status == SyncFileItem::Conflict && !Utility::isConflictFile(item->_file) {
        return false;
    }

    return true;
}
`

Reply:
@allexzander  I find it more descriptive with the variable names. Then it's even readable without the comments:) With you approach it's just one line less. However, I like your comments about the conditionals. I updated it and changed the wording a bit.